app/src/main/java/com/ncnf/event/EventRelevanceCalculator.java

```diff
@@ -1,34 +1,31 @@
 package com.ncnf.event;
 
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
 
 public class EventRelevanceCalculator {
```

Comment:
Can be renamed to SocialRelevanceCalculator if it can be used for any Social object.

Reply:
Also fixed !